server/datastore/mysql/migrations/tables/20211221110132_AddHostEmailsTable.go

```diff
@@ -0,0 +1,36 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20211221110132, Down_20211221110132)
+}
+
+func Up_20211221110132(tx *sql.Tx) error {
```

Comment:
Note that I updated the migration's date as another migration (with a later date) got merged in the meantime. To keep in mind if this is not merged today.

Reply:
Oh nevermind, I missed that the PR had been approved, will merge once CI passes.